package.json

```diff
@@ -26,19 +26,20 @@
   "dependencies": {
     "bluebird": "^2.9.34",
     "extend": "^3.0.0",
-    "jayschema": "^0.3.1",
-    "js-yaml": "^3.4.0",
-    "matrix-appservice": "^0.3.4",
+    "is-my-json-valid": "^2.19.0",
+    "js-yaml": "^3.12.0",
+    "matrix-appservice": "matrix-org/matrix-appservice-node#73839c6077b92b4cd0d52ec8dee599a96142d3b3",
```

Comment:
I'd rather us wait for the package to be released before we merge this PR

Reply:
Seems fair